engine/runtime-integration-tests/src/test/scala/org/enso/compiler/test/CompilerTest.scala

```diff
@@ -151,23 +151,23 @@ trait CompilerRunner {
       */
     def asMethod: definition.Method = {
       new definition.Method.Explicit(
-        Name.MethodReference(
```

Comment:
How is this change related to diagnostic storage?

Reply:
During the compilation, the `Explicit` node is always created from the `Binding` copying the accompanying diagnostic, location, etc. The constructor was updated to DRY it.